examples/run_CPMG.py

```diff
@@ -10,116 +10,132 @@
 from pyspecdata import *
 from numpy import *
 import SpinCore_pp
-from SpinCore_pp.ppg import run_cpmg
+from SpinCore_pp import prog_plen
+from SpinCore_pp.ppg import generic
 import os
 from datetime import datetime
 import h5py
-raise RuntimeError("This pulse proram has not been updated.  Before running again, it should be possible to replace a lot of the code below with a call to the function provided by the 'generic' pulse program inside the ppg directory!")
+from Instruments.XEPR_eth import xepr
 
-fl = figlist_var()
 # {{{importing acquisition parameters
 config_dict = SpinCore_pp.configuration("active.ini")
-nPoints = int(config_dict["acq_time_ms"] * config_dict["SW_kHz"] + 0.5)
+config_dict["SW_kHz"] = 75e6/round(75e6/config_dict["SW_kHz"]/1e3)/1e3
+nPoints = int(config_dict["echo_acq_ms"] * config_dict["SW_kHz"] + 0.5)  # per echo
+my_exp_type = "ODNP_NMR_comp/Echoes"
+target_directory = getDATADIR(exp_type=my_exp_type)
+config_dict["echo_acq_ms"] = nPoints / config_dict["SW_kHz"]
```

Comment:
actually, can we just supply a function (stored somewhere in the package) that would be used as follows?
```
nPoints, config_dict["SW_kHz"], config_dict["echo_acq_ms"] = get_integer_sampling_intervals(config_dict["SW_kHz"], config_dict["echo_acq_ms"])
```
where `get_integer_sampling_intervals` does lines 22, 23, and 26 above?

Reply:
implemented simple_fns.py in SpinCore_pp

Reply:
@guinneak198 to be specific -- notice how the parameters and returns for this function are different vs what you implemented.  This would be less  lines of code, and you can switch between the different acq variables seemlessly.